main.go

```diff
@@ -52,6 +53,8 @@ func init() {
 	utilruntime.Must(datadoghqv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(edsdatadoghqv1alpha1.AddToScheme(scheme))
 	// +kubebuilder:scaffold:scheme
+
+	klog.Configure(ctrl.Log.WithName("klog"))
```

Comment:
Should be put klog configuration first in the init() ? if never the `utilruntime` also log something

Reply:
Done !